_source/_posts/2019-09-18-build-a-simple-dotnet-core-app-in-docker.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: blog_post
+title: "Build a Simple .NET Core App on Docker"
+author: charlieholland
+description: "Learn how to containerize your secure .NET Core applications with Docker, the Okta way."
+tags: [ dotnetcore, docker, dotnet, .netcore, csharp, containerization, containers ]
+tweets:
+- "Ever wonder what you can with #Docker + #AspNetCore? Check it out! →"
+- "Hey .NET devs, we've got a simple #Docker tutorial for your #AspNetCore apps! →"
+- "Docker + .NET Core = <3 →"
+image: blog/featured/okta-dotnet-mouse-down.jpg
+---
+
+Wouldn't it be great if stuff just worked? Especially in the ever-changing world of software. Chasing dependency issues and debugging arcane operating system errors - not a good use of time.
+
+One important aspect of "stuff just works" -- reliability. Recently, the software community has made strides in test-driven development and continuous integration processes to drive up quality, and of course, that improves reliability. But it can only go so far. Operating systems perform many functions and incorporate many features, so running them reliably is a huge task. The core secret to improving reliability is fewer moving parts. In this post, I'll cover how you can use Docker to run your application in an isolated, minimal environment with fewer moving parts.
+
+## Sample App Dependencies: ASP.Net Core and Docker Packages
+
+To build an app in Docker, first we need an app to Dockerize. This tutorial uses the  ASP.Net Core project from a previous blogpost on [Adding Login to Your ASP.NET Core MVC App](blog/2018/10/29/add-login-to-you-aspnetcore-app).
+That blogpost shows you how to build a simple .Net Core application that uses Okta for Identity Management. You should work through that blog post, or at very least read it and clone the repo.
+
+You'll also need:
```

Comment:
I would change this to match the Prerequisites block like https://developer.okta.com/blog/2019/09/11/angular-mongodb has.